geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -357,6 +357,27 @@ long zcount(SortedSetScoreRangeOptions rangeOptions) {
     return addLimitToRange(rangeOptions, withScores, minIndex, maxIndex);
   }
 
+  long zlexcount(SortedSetLexRangeOptions lexOptions) {
+    // Assume that all members have the same score. Behaviour is unspecified otherwise.
+    double score = scoreSet.get(0).score;
+
+    AbstractOrderedSetEntry minEntry = new MemberDummyOrderedSetEntry(lexOptions.getMinimum(),
```

Comment:
consider extracting these two blocks into getMinIndex(lexOptions, score) and getMaxIndex(lexOptions, score). These new methods could also be used by zrangebylex

Reply:
If I can get the ZREVRANGEBYSCORE PR pushed up, we can use the helper methods I added there.

Reply:
@ringles Could you point me at the methods you're referring to?